aspnetcore/fundamentals/middleware/extensibility/sample/Extensions/AspNetCoreExtensions.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Globalization;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace MissingDIExtensions
+{
+    public static class AspNetCoreExtensions
```

Comment:
This is *way* too much code for this topic. I think we want to show a minimal example of using the interface with the built in container and a very simple way of using it with something like ninject/simple injector. 

This seems to have copied all of the code from that github repo talking about non conforming containers.

Reply:
True ... not having ever implemented Ninject (or any other container for that matter), I was going by the documented sample that I had at hand. It wasn't apparent which parts were optional for a minimal configuration.

I'll see if I can whittle it down to the bare essentials.

Reply:
@davidfowl  it might be best if you assigned one of your dev's to come up with a minimal sample. There's a reason we're not devs 😸

Reply:
@Rick-Anderson Let me take a crack at this. The problem here was simply that I implemented *too much* code. The sample here works just fine ... it was just overkill to use Ninject like it's going to be implemented in a full project. I may admit defeat later, but I'll be looking into this within just a few hours, and we'll see soon how paring it down goes.